### ISSUE TITLE
scalers.sh includes bad upscalers by default

### ISSUE THREAD

**Issue by haasn** [haasn/cms#3]
Some of the scale settings, in particular scale=custom, but also lesser useful aliases for existing things (like scale=bilinear which is the same as scale=triangle, and scale=box which is the same as scale=nearest with the wrong radius set by default) should probably be blacklisted from the default list of scalers.

@Shudouken, do you know how to add this easily? A hard-coded list like "custom box bilinear" might be all that's needed.

**Comment by Shudouken** [comment]
```
blacklist="custom bilinear box"
for item in $blacklist; do
  scalers=$(echo "$scalers" | sed -e "s/$item//")
done
```

This should do it, just place it before the main for loop, it would probably make more sense to add these in hardcoded rather than calling the script with something like -b "bilinear" all the time

**Comment by haasn** [comment]
I was thinking it should just apply to the default -s setting (ie. default is to use all available scalers except X).

**Comment by haasn** [comment]
I just realized that's literally the same thing, just at a different place in the code (before the options parsing).

Merged in de427b1, thanks.
